pages/linux/dget.md

```diff
@@ -0,0 +1,16 @@
+# dget
+
+> Download Debian packages.
+> More information: <https://manpages.debian.org/buster/devscripts/dget.1.en.html>.
```

Comment:
Is there a URL that's independent of the current release  of Debian (e.g. replacing `buster` with `stable` or something?)

Reply:
We can use a redirect like this: https://manpages.debian.org/dget
`stable` also seems to redirect correctly: https://manpages.debian.org/stable/devscripts/dget.1.en.html

Reply:
Sounds good to me! Let's replace the URL with that then. That  way we don't have to keep updating the URL in the tldr page :-)

Reply:
I guess it would need to be changed on other pages too:
```
pages/common/git-subtree.md:> More information: <https://manpages.debian.org/testing/git-man/git-subtree.1.en.html>.
pages/common/makepasswd.md:> More information: <https://manpages.debian.org/stretch/makepasswd/makepasswd.1.en.html>.
pages/linux/a2disconf.md:> More information: <https://manpages.debian.org/buster/apache2/a2disconf.8.en.html>.
pages/linux/a2dismod.md:> More information: <https://manpages.debian.org/buster/apache2/a2dismod.8.en.html>.
pages/linux/a2dissite.md:> More information: <https://manpages.debian.org/buster/apache2/a2dissite.8.en.html>.
pages/linux/a2enconf.md:> More information: <https://manpages.debian.org/buster/apache2/a2enconf.8.en.html>.
pages/linux/a2enmod.md:> More information: <https://manpages.debian.org/buster/apache2/a2enmod.8.en.html>.
pages/linux/a2ensite.md:> More information: <https://manpages.debian.org/buster/apache2/a2ensite.8.en.html>.
pages/linux/a2query.md:> More information: <https://manpages.debian.org/buster/apache2/a2query.1.en.html>.
pages/linux/check-support-status.md:> More information: <https://manpages.debian.org/buster/debian-security-support/check-support-status.1.en.html>.
pages/linux/lastcomm.md:> More information: <https://manpages.debian.org/stable/acct/lastcomm.1.en.html>.
pages/linux/reportbug.md:> More information: <https://manpages.debian.org/buster/reportbug/reportbug.1.en.html>.
```